### PR TITLE
Added ability to customize tab sizes (d_tabw and d_tabh) + function for multiple equally sized bins

### PR DESCRIPTION
**Pull request by Asger1002**
Added a function ('cutEqualBins') for cutting several identical bins, with their size divided along the specified length. Unlike the 'cutEqual' function, one can specify the position of the bins as well. See point '1' in attached image.

Added two parameters in 'cut' (and the modules it calls) for specifying tab width and tab height. Tab width helps with some bins, where a narrower tab can make it easier to grab f.x. long screws from a bins, as in '3' in the attached image. Tab height refers to how much the tab 'sticks out', so it fits either wider labels, or narrower labels without wasting space ('4' in image).

Attached image shows:
1. A line of equally sized bins
2. A 1.5 wide bin, with default label size, which can be difficult to reach into.
3. A 1.5 wide bin with a label size of 35, easier to reach into
4. A 1.5 wide bin, with a default-width, centered label that has a height of 5mm.

I've done my best to check the functions for bugs and it all seems to be working without any issues. With that said, I've only used Openscad for a few days now, and never opened a pull request, so if something is missing or I haven't followed protocol, please let me know and I'll do my best to fix it.

numbered_illustration
code_for_numbered_illustration

### CODE REVIEW

File: gridfinity-rebuilt-utility.scad [hunk 1]
```diff
@@ -301,9 +335,9 @@ module cut_move_unsafe(x, y, w, h) {
     children();
 }
 
-module block_cutter(x,y,w,h,t,s) {
+module block_cutter(x,y,w,h,t,s,tab_width=d_tabw,tab_height=d_tabh) {
 
-    v_len_tab = d_tabh;
+    v_len_tab = tab_height;
     v_len_lip = d_wall2-d_wall+1.2;
     v_cut_tab = d_tabh - (2*r_f1)/tan(a_tab);
```
Comment (Asger1002) [inline gridfinity-rebuilt-utility.scad:342]: This should be changed to
`v_cut_tab = tab_height - (2*r_f1)/tan(a_tab);`

A small line remained when using a smaller tab height. Now they look perfect, as far as i can see

Comment (Asger1002) [reply gridfinity-rebuilt-utility.scad:342]: I would do it, but i don't know how after starting the pull request. Sorry!

**Comment by Ruudjhuu** [comment]
Nice job, I like the modularity. 

Your commits all have the same message, could you give them descriptive messages or squash them and 1 descriptive message? 

If you don't know how, google interactive rebase.

**Comment by Asger1002** [comment]
> Nice job, I like the modularity.
> 
> Your commits all have the same message, could you give them descriptive messages or squash them and 1 descriptive message?
> 
> If you don't know how, google interactive rebase.

Thank you for your hint on the rebase, I think i've done that now; and squashed the commits together with 1 message describing what was done. I've pushed a new commit too, fixing the line i mentioned and adding a comment describing the functionality/inputs of the new CutEqualBins function, so it should be in line with the other user-facing functions.

If anything else is missing or not up to standard I'll be happy to fix it!

**Comment by Ruudjhuu** [comment]
If you make the test green, I can merge this. Great work!

**Comment by Asger1002** [comment]
> If you make the test green, I can merge this. Great work!

The test mentioned something about failure to remove trailing whitespaces. I've removed all of these from the code snippet it showed, but I cannot find a way for me to re-run the test for whether this works; it seems I need approval?

Apologies for using your time on this; I am simply too new to using github and proper development workflows.

**Pull request merged**